src/constants/messages.ts

```diff
@@ -8,8 +8,11 @@ export const messages = {
   getTransferFailed: 'Unable to get transfer',
   invalidEndpoint: 'The specified endpoint is not a valid URL',
   loadingProtocol: 'Launching IBM Aspera for Desktop',
+  modifyTransferFailed: 'Unable to modify transfer',
   notValidTransferSpec: 'The supplied transferSpec is not valid',
   removeTransferFailed: 'Unable to remove transfer',
+  resumeTransferFailed: 'Unable to resume transfer',
+  sdkAlreadyInitialized: 'The Desktop SDK has already been initialized',
```

Comment:
Since this is new can you call it `Browser SDK`

Reply:
Done!